### DISCUSSION TITLE
Snap Panel is awesome, so of course I now have some requests…

### DISCUSSION THREAD

**Discussion by RyanRoberts** [rxhanson/RectanglePro-Community#101] [Ideas]
I've wanted this Snap Panel functionality for a while but never thought to ask, thanks for adding it!

Would you consider adding the ability to trigger it via a shortcut? You'd then click the position you want rather than drag a window to it. 

[Custom positions use a generic icon](https://media.yep.digital/AQ8x14) which is not very descriptive of the action, will you add the ability to customise these somehow?

Thanks

**Comment by rxhanson** [comment]
Thanks for the quick feedback! This is the kind of feedback I was hoping to get, since it's the initial addition of the feature and I'm testing the waters to see if it's something that people want, and where to focus attention.

> Would you consider adding the ability to trigger it via a shortcut? You'd then click the position you want rather than drag a window to it.

Makes sense to me. I guess to close it I could just detect if the user clicks anywhere outside of it, or an additional press of the shortcut. I was also considering making a "window throw"-esque modifier only shortcut for it, which would make it feel pretty similar to [Charmstone](https://charmstone.app) but for window management.

> [Custom positions use a generic icon](https://media.yep.digital/AQ8x14) which is not very descriptive of the action, will you add the ability to customise these somehow?

The easiest would be to just use the first letter or two letter abbreviation of the custom name, so as to retain the same size/position and not add a bunch of text. Does that seem sufficient?

I do have a number of things lined up before I'll make it back to this, so if any more ideas come up or if anyone else wants to chime in, feel free!

**Reply by RyanRoberts** [reply]
That all sounds good to me 👍
